Allowed webhooks to be given as a list of strings

Math0424 opened the pull request:
Changed the webhook url to a list of url's

Math0424 committed:
Allowed webhooks to be given as a list of strings

Gwindalmir reviewed (changes requested):
I wish you'd sent this before I made the most recent release, but I don't think too many people are using the feature yet, so I won't worry about backwards compatibility here. 😄 

However, I wanted to ask you last time, but missed the opportunity: since you've submitted this one now, would you mind creating some tests for this?
You should be able to create a new file in Tests\Common, following the same style as [Tests/Common/IgnoreFileTests.cs](https://github.com/Gwindalmir/SEWorkshopTool/blob/master/Tests/Common/IgnoreFileTests.cs) or [UpdaterTests.cs](https://github.com/Gwindalmir/SEWorkshopTool/blob/master/Tests/Common/UpdaterTests.cs).
They don't need to be complicated (and shouldn't be). Basically, create a method that calls your discord webhook method (you might need to refactor some of the existing code to do so).

I don't know how webhooks works, so I don't have the knowledge to write this myself and I don't want to risk breaking this in the future.

I don't know if it _will_ be possible, but if you can, that would be great.

Math0424 committed:
+ Added test cases
+ Refactored webhook for tests
+ Changed input name
+ Removed requested code
+ Fixed potential crash
